mtags/src/main/scala-3/scala/meta/internal/pc/InferredTypeProvider.scala

```diff
@@ -251,4 +255,32 @@ final class InferredTypeProvider(
     end match
   end inferredTypeEdits
 
+  private def namePos(text: String, tree: untpd.NameTree)(using
+      Context
+  ): SourcePosition =
+    val realName = tree.name.stripModuleClassSuffix.lastPart.toString.toList
```

Comment:
Could you provide some example how `tree.name.stripModuleClassSuffix` and `tree.name.stripModuleClassSuffix.lastPart` differs from each other?

Reply:
@kpodsiad  I don't know. I just copied this line from from compiler impl for `namePos`

Reply:
Ok, adding info that `this line from from compiler impl for namePos` will also be helpful for future us.

Reply:
ok, just removed `lastPart` call. Tests pass without it.